GOVERNANCE.md

```diff
@@ -0,0 +1,87 @@
+maintainers.
+
+Every Maintainer is listed in the top-level [OWNERS](https://github.com/coredns/coredns/OWNERS)
+file, with their Github handle and an (possible obfuscated) email address. Every one in the
+`reviewers` list is a Maintainer.
+
+A Maintainer is also listed in a plugin specific OWNER file.
+
+A Maintainer should be a member of `maintainer@coredns.io`, although this is not a hard requirement.
+A Maintainer that hasn't been active in the CoreDNS repository for 12 months is considered inactive.
+
+## Becoming a Maintainers
+
+On successful completion (it was merged) of a (large) pull request, any current maintainer can reach
+to the person behind the pull request and ask them if they are willing to become a CoreDNS
+maintainer.
+
+## Changes in Maintainership
+
+If a Maintainer feels she/he can not fulfill the "Expectations from Maintainers", they are free to
+step down.
+
+The CoreDNS organization will never forcefully remove a current Maintainer, unless a maintainer
+fails to meet the principles of CoreDNS community.
+
+
+## Other Projects
+
+The CoreDNS organization is open to receive new sub-projects under its umbrella. To accept project
+into the __CoreDNS__ organization, it has to met the following criteria:
+
+- Licensed under the terms of the Apache License v2.0.
+- Related to one or more scopes of CoreDNS ecosystem:
+  - CoreDNS project artifacts (website, deployments, CI, etc ...)
+  - External plugins
+  - other DNS processing related
+- Be supported by a Maintainer.
+
+The submission process starts as a Pull Request or Issue on the
+[coredns/coredns](https://github.com/coredns/coredns) repository with the required information
+mentioned above. Once a project is accepted, it's considered a __CNCF sub-project under the umbrella
+of CoreDNS__
+
+## Decision making process
+
+Decisions are build on consensus between maintainers.
+Proposal and ideas can either be submitted for agreement via an github issue or by sending an email to `maintainer@coredns.io`
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out between the persons involved.
+If a dispute cannot be decided independently, the maintainers can be called in to decide an issue.
+If the maintainers themselves cannot decide an issue, the issue will be resolved by voting.
+
+For formal votes, a specific statement of what is being voted on, and in which delay (a suitable amount of time),
+should be added to the relevant github issue or PR, and a link to that issue
+or PR sent to `maintainer@coredns.io`.
+
+Maintainers should indicate their yes/no vote (or respectively +1/-1) on that issue or PR,
+and after the delay is expired, the votes will be tallied and the outcome noted.
+
+A 2/3 majority vote is needed for the statement to be approved.
+
+Each maintainer weighs one vote.<br>
+Miek Gieben (@miekg), as the historical owner of CoreDNS, weighs two votes.
```

Comment:
```suggestion
Miek Gieben (@miekg), as the owner of CoreDNS, weighs two votes.
```

Reply:
I am not sure how to define the owner of coreDNS - and if there is one, since we moved to cncf and it is opensource project. 
From point of view of Github there are several owners.

That is why I prefer the "historical owner" that show, whatever owners are defined today, that the project were started and grown up by Miek.

I propose to keep "historical owner" for now.

Reply:
unless there is a more appropriate way to say the same thing ?